### PR TITLE
Input helper support in Client

### PR DESCRIPTION
**Pull request by helto4real**
## Proposed change

This is the first PR of several that will add support to create, delete and list Helpers from HA.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (which adds functionality to an existing integration)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] Local tests pass. **Your PR cannot be merged unless tests pass**
- [x] There is no commented out code in this PR.
- [x] I have followed the [development checklist][dev-checklist]
- [x] The code compiles without warnings (code quality chek)
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/net-daemon/docs

### CODE REVIEW

File: src/Client/NetDaemon.HassClient/Internal/HomeAssistant/Commands/CreateInputBooleanHelperCommand.cs [hunk 1]
```diff
@@ -1,9 +1,9 @@
 using System.Text.Json.Serialization;
 using NetDaemon.Client.HomeAssistant.Model;
 
-namespace NetDaemon.Runtime.Internal.Model;
+namespace NetDaemon.Client.Internal.HomeAssistant.Commands;
```
Comment (FrankBakkerNl) [inline src/Client/NetDaemon.HassClient/Internal/HomeAssistant/Commands/CreateInputBooleanHelperCommand.cs:4]: Keeping 'internal' in the namespace is odd when making the classes public

Comment (helto4real) [reply src/Client/NetDaemon.HassClient/Internal/HomeAssistant/Commands/CreateInputBooleanHelperCommand.cs:4]: My bad....

File: src/Client/NetDaemon.HassClient/Common/HomeAssistant/Model/InputBooleanHelper.cs [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ﻿using System.Text.Json.Serialization;
 
-namespace NetDaemon.Runtime.Internal.Model;
+namespace NetDaemon.Client.HomeAssistant.Model;
 
-internal record InputBooleanHelper
+public record InputBooleanHelper
 {
-    [JsonPropertyName("name")] public string Name { get; init; } = string.Empty;
+    [JsonPropertyName("name")] public string Name { get; init; } = String.Empty;
```
Comment (FrankBakkerNl) [inline src/Client/NetDaemon.HassClient/Common/HomeAssistant/Model/InputBooleanHelper.cs:7]: In stead of using a default value you can mark them required if they should always be specified, if not they should be nullable

Comment (helto4real) [reply src/Client/NetDaemon.HassClient/Common/HomeAssistant/Model/InputBooleanHelper.cs:7]: Yea good point. I forgot about the required keyword

**Review by FrankBakkerNl — APPROVED** [review]

**Pull request merged**